tasks/sds.py

```diff
@@ -0,0 +1,38 @@
+import os
+import sys
+import tempfile
+
+from invoke import task
+
+from tasks.rtloader import get_dev_path
+
+is_windows = sys.platform == "win32"
+is_darwin = sys.platform == "darwin"
+
+
+@task
+def build_library(ctx):
+    """
+    Build the SDS shared library
+    """
+    if is_windows:
+        print("not supported")
```

Comment:
```suggestion
        print("Not building the SDS library as it's not supported on Windows.", file=sys.stderr)
```

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could you please give more context in the error message ? I fear that in its current state the message will be lost in the trace.